client/hooks/InterfaceFieldsToTransmit/MeinBerlinProcedureFields.vue

```diff
@@ -0,0 +1,353 @@
+<template>
+  <div>
+    <h3>
+      {{ Translator.trans('mein.berlin.interface') }}
+    </h3>
+
+    <component
+      :is="demosplanUi.DpInlineNotification"
+      v-if="!isCheckingBerlinOrgaId && (isProcedureTransmitted || !hasBerlinOrgaId)"
+      :message="isProcedureTransmitted
+        ? Translator.trans('mein.berlin.procedure.already.transmitted')
+        : Translator.trans('mein.berlin.organisation.id.missing.transmission.not.possible')"
+      class="mb-4"
+      type="info"
+    />
+
+    <component
+      :is="demosplanUi.DpCheckbox"
+      id="interfaceFieldsToTransmit-checkbox"
+      v-model="isInterfaceActivated"
+      :checked="isInterfaceActivated"
```

Comment:
I think the `checked` attribute is redundant here because you are already using `v-model`